proto/panacea/oracle/v2alpha2/oracle.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package panacea.oracle.v2alpha2;
+
+option go_package = "github.com/medibloc/panacea-core/x/oracle/types";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+// OracleStatus enumerates the status of oracle.
+enum OracleStatus {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // ORACLE_STATUS_UNSPECIFIED
+  ORACLE_STATUS_UNSPECIFIED = 0;
```

Comment:
Does this status mean that the Oracle is voting?

Reply:
And I wonder if this state is maintained even in case of registration failure.

Reply:
> Does this status mean that the Oracle is voting?

When we define enum in proto, we are forced to use index 0 as a status of `UNSPECIFIED`. If not, proto lint fail.
As you can see in all the status enum, 0 index is used for `UNSPECIFIED`.

> And I wonder if this state is maintained even in case of registration failure.

This status of `OracleStatus` is for registered oracle. As I understand, when oracle is failed to be registered, the oracle would not be stored in `Oracle` KV store. But it may be stored in `OracleRegistration` KV store with status of `FAILURE`. Do I understand correctly?

Reply:
> > Does this status mean that the Oracle is voting?
> 
> When we define enum in proto, we are forced to use index 0 as a status of `UNSPECIFIED`. If not, proto lint fail. As you can see in all the status enum, 0 index is used for `UNSPECIFIED`.

Aha. I got it. Thank you.

Reply:
> > And I wonder if this state is maintained even in case of registration failure.
> 
> This status of `OracleStatus` is for registered oracle. As I understand, when oracle is failed to be registered, the oracle would not be stored in `Oracle` KV store. But it may be stored in `OracleRegistration` KV store with status of `FAILURE`. Do I understand correctly?

Ah, I thought I would also store it in the Oracle KV store, but from what you said, it doesn't seem necessary.
I want it to be like this now.

Reply:
Thanks for asking!
I also thought as you did, but I realized when defining some statuses.